modules/influxdb/build.gradle

```diff
@@ -1,8 +1,16 @@
 description = "Testcontainers :: InfluxDB"
 
+ext {
+    influxdbV1ClientVersion = '2.21'
+    influxdbV2ClientVersion = '3.1.0'
+}
+
```

Comment:
Please could we return to the original one-string form for specifying these dependencies? It's the convention we use throughout, and works well with Dependabot.